pandas/core/tools/timedeltas.py

```diff
@@ -21,18 +21,18 @@ def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     Convert argument to timedelta.
 
     Timedeltas are absolute differences in times, expressed in difference
-    units e.g. days, hours, minutes, seconds. This method converts an argument
+    units e.g. (days, hours, minutes, seconds). This method converts an argument
     from a recognized timedelta format / value into a Timedelta type.
 
     Parameters
     ----------
-    arg : str, timedelta, list, tuple, 1-d array, or Series
+    arg : str,  timedelta,  list,  tuple,  1-d array,  or Series
```

Comment:
the missing spaces were down, in the `(D,h,m,s...)`, which should be `(D, h, m, s,...)`.

Just leave a space after the commas here.